content/en/developers/write_agent_check.md

```diff
@@ -160,5 +160,5 @@ class LSCheck(AgentCheck):
 [2]: https://github.com/DataDog/integrations-extras
 [3]: http://app.datadoghq.com/account/settings#agent
 [4]: /help
-[5]: https://datadog-checks-base.readthedocs.io/en/latest/datadog_checks.checks.html#datadog_checks.base.checks.base.AgentCheck
+[5]: https://datadoghq.dev/integrations-core/base/api/#datadog_checks.base.checks.base.AgentCheck
```

Comment:
Open question: Can we point to datadoghq.dev?

Reply:
@jeremy-lq  ^

Reply:
As the content is public yes :)